toptal/picasso
# refactor: [FX-2315] make colors consistent

[FX-2315]

### Description

Reverting naming of colors back to concrete colors for `Tag`, `Tag.Rectangular`, `Indicator`

We are not ready yet to switch to abstract color names in the whole application,
maybe we will come back to it when we will need to make Picasso themeable.

To revert most of the cases you can use [codemon for v16.0.0](https://github.com/toptal/picasso/tree/master/packages/picasso-codemod#v1600)

### How to test
- there should be no changes in visual tests
- [Tag story](https://picasso.toptal.net/fx-2315-make-colors-consistent/?path=/story/components-tag--tag) should look as before
- [Indicator story](https://picasso.toptal.net/fx-2315-make-colors-consistent/?path=/story/components-indicator--indicator) should look as before
- [x] alpha package codemon tested on Staff portal

### Review


<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and
1. You have no extra requests.
2. You have optional requests.
   1. Add `nit: ` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because
1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:
1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>


[FX-2315]: https://toptal-core.atlassian.net/browse/FX-2315?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
